lib/ox/element.rb

```diff
@@ -150,7 +150,18 @@ def locate(path)
     # _raise_ [NoMethodError] if no match is found
     def method_missing(id, *args, &block)
       has_some = false
-      ids = id.to_s
+      ids = id.to_s.sub('_', ':') # allows easy API with namespaces
+      
+      if ids[-1] == '!' # return array of elements matching ids
```

Comment:
I tried monkey-patching the `each` in, but after `element` has resolved, you would need to call something like `self.parent.nodes.select{|n| n.name == self.name}` in `each` on `element` to get it to work (as far as I can tell).